apps/emqx_durable_storage/src/emqx_ds_storage_layer.erl

```diff
@@ -586,6 +589,24 @@ init({ShardId, Options}) ->
     commit_metadata(S),
     {ok, S}.
 
+format_status(#s{shard_id = ShardId, db = DB, cf_refs = CFRefs, schema = Schema, shard = Shard}) ->
```

Comment:
Hm, I thought that the argument of `format_status/1` callback is not the state itself but something that contains it among other things.
https://www.erlang.org/doc/man/gen_server#type-format_status

Reply:
Thanks, fixed.